### ISSUE TITLE
Get review of a11y

### ISSUE THREAD

**Issue by Mardak** [mozilla/activity-stream#2014]
Split from #1978. We have a list of ideas of what might be a11y-impacting, so it would be good to get a second set of eyes to make sure we didn't miss anything as well as provide guidelines of how issues should be fixed.

**Comment by Mardak** [comment]
No formal review step needed, so we'll untrack this issue.
